src/samples/flow/oic/light-client.json

```diff
@@ -0,0 +1,11 @@
+{
+    "nodetypes": [
+        {
+            "type": "oic/client-core-brightlight",
+            "name": "Light",
+            "options": {
+                "hwaddr": "\"08:00:27:ab:45:32\""
```

Comment:
This was automatically added by the python script - most probably it had quotes on the config line.